datafusion/physical-expr/Cargo.toml

```diff
@@ -67,6 +67,7 @@ regex = { version = "1.8", optional = true }
 sha2 = { version = "^0.10.1", optional = true }
 unicode-segmentation = { version = "^1.7.1", optional = true }
 uuid = { version = "^1.2", features = ["v4"] }
+edit-distance = "2.1.0"
```

Comment:
I think we can avoid this new dependency by either reusing our existing implementation (see below)